cogs/Announce.py

```diff
@@ -0,0 +1,418 @@
+#!/usr/bin/env python
+
+"""
+Koala Bot Announce feature
+Created by: Bill Cao
+"""
+
+# Built-in/Generic Imports
+import asyncio
+
+# Libs
+import math
+from typing import Optional, Tuple
+
+import discord
+from discord.ext import commands
+from utils.KoalaUtils import extract_id
+from utils import KoalaColours
+import time
+
```

Comment:
Add logging everywhere important where an error could occur (look at TwitchAlert, RFR and Verify which I think have alright logging)
```suggestion
import logging
logging.basicConfig(filename='Announce.log')
```